release-notes.md

```diff
@@ -36,6 +36,40 @@ This new profile is intended for iterative development versus the path to produc
 * Enabled multiple cluster support for Application Live View.
 * Split Application Live View components into three packages with new package reference names.
 
+#### Tanzu CLI - Apps plugin
+* `workload create/update/apply`:
+  * Accept `workload.yaml` from stdin (via `--file -`).
+  * Enable providing `spec.build.env` values (via new `–build.env` flag).
+  * When `--git-url` and `--git-tag` are provided, `git-branch` is not also required.
```

Comment:
We don't have a `--git-url` flag. I think you mean that we can create workload specifying either `--git-tag`, `--git-branch` or both.

Reply:
good catch!

fixed in this commit https://github.com/heyjcollins/docs-tap/commit/16e2128b0c914cda9e5fe7522ebbdb7193a89557